metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_schema_gen.py

```diff
@@ -176,6 +176,7 @@ def __init__(
         aggregator: Optional[SqlParsingAggregator],
         snowsight_url_builder: Optional[SnowsightUrlBuilder],
         fetch_views_from_information_schema: bool = False,
+        is_temp_table: Callable[[str], bool] = lambda _: False,
```

Comment:
maybe we should call this `is_temp_table_fn` and be optional `Optional[Callable[str]]` and not `Callable[[str], bool]`